Orange/widgets/evaluate/owcalibrationplot.py

```diff
@@ -58,13 +60,51 @@
 )]
 
 
+class ParameterSetter(Setter):
+    initial_settings = OrderedDict({
+        Setter.LABELS_BOX:
+            OrderedDict({name: values for name, values in (
```

Comment:
Perhaps you already told me and I forgot -- why do you explicitly need to use `OrderedDict` here?

Reply:
> why do you explicitly need to use `OrderedDict` here?
To assure ordering in the visual settings dialog. It is not really necessary.

Reply:
Pure `dict` is already ordered from (C)Python 3.6 onwards.

Reply:
Is it? Ok, I'll remove the OrderedDict.